### PR TITLE
Use ListValueNotifier for filtered data

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Please review with hide whitespace diffs.

### CODE REVIEW

File: packages/devtools_app/lib/src/network/network_controller.dart [hunk 1]
```diff
@@ -384,49 +384,53 @@ class NetworkController
...
+            return false;
           }
-          return false;
-        }
-        _checkForError(r);
-        return true;
-      }).toList();
+          _checkForError(r);
+          return true;
+        }).toList());
```
Comment (jacob314) [inline packages/devtools_app/lib/src/network/network_controller.dart:433]: the `toList()`
isn't needed.

File: packages/devtools_app/lib/src/network/network_controller.dart [hunk 2]
```diff
@@ -384,49 +384,53 @@ class NetworkController
     if (filter == null) {
       final requests = List<NetworkRequest>.from(_requests.value.requests);
```
Comment (jacob314) [inline packages/devtools_app/lib/src/network/network_controller.dart:385]: you can avoid this List.from

**Review by jacob314 — APPROVED** [review]
lgtm
This change is fine and it is nice to be consistent using ListValueNotifier but in this case there isn't a huge amount of benefit as I don't see any cases where elements are being incrementally added to the list.

File: packages/devtools_app/lib/src/network/network_controller.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/network/network_controller.dart:433]: without toList() an error like `'WhereIterable<LogData>' is not a subtype of type 'List<LogData>'` is thrown

**Pull request merged**